### DISCUSSION TITLE
`fzf-lua` is not displayed properly

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [LazyVim/LazyVim#4070] [Q&A] [answered]
After `<leader>ff`, press `F4` to close the preview, and the effect will be as follows.

The same is true for other `fzf-lua` related operations.

![image](https://github.com/user-attachments/assets/cb66786f-762f-4d9d-ba7d-b3fda3bac698)

**Comment by dpetka2001** [comment]
Probably something with fzf-lua on Windows. Don't see that on Linux. Maybe try a different terminal like `wezterm` to see if it's a terminal issue or not?

**Reply by iamxiaojianzheng** [reply]
I tried wezterm and had the same problem.

**Reply by iamxiaojianzheng** [reply]
As far as I can recall, this was never the case before.

**Reply by dpetka2001** [reply]
Then maybe it's a fzf-lua thing on Windows? I don't see this artifact on Linux. 

[Screencast 2024-07-16 12:20:16.webm](https://github.com/user-attachments/assets/a55c8147-3514-4463-b876-0de5c79dfb8e)

**Reply by iamxiaojianzheng** [reply]
emmm, all right. That's a strange question.

**Comment by iamxiaojianzheng** [comment]
I accidentally found the reason for this problem, because of the fzf itself.

This happens when fzf is upgraded to `0.54.0`. When I went down to `0.53.0`, It went back to normal.

**Reply by dpetka2001** [reply]
Just upgraded to 0.54 (was on 0.53 before) and I still can't reproduce on Linux. It's exactly the same as on the screencast that I previously attached. Maybe only a Windows thing?

**Reply by iamxiaojianzheng** [reply]
Perhaps this is a problem that only occurs under Windows. 😢

**Reply by folke** [reply]
You should open an issue in the fzf-lua repo. This has nothing to do with LazyVim.

**Reply by iamxiaojianzheng** [reply]
ok, I have raised this issue in fzf-lua.

**Comment by iamxiaojianzheng** [comment, marked as answer]
This issue no longer occurs when fzf is upgraded to 0.54.2.
